ref_ts/views.py

```diff
@@ -221,7 +221,7 @@ def download_refts_resource_files(request, shortkey, *args, **kwargs):
             return response
 
         path = "bags/" + str(shortkey) + ".zip"
-        response_irods = download_bag_from_irods(request, path)
+        response_irods = download_bag_from_irods(request, path, use_async=False)
```

Comment:
I am OK with merging it for now. I suggest that we remove returning html content in any view function response as part of needed additional work on the api side.